authorizer-app/src/app/components/rest-source-authorization/rest-source-user-list.component.html

```diff
@@ -40,11 +40,11 @@
       </ng-container>
       <ng-container matColumnDef="userId">
         <th mat-header-cell *matHeaderCellDef mat-sort-header> User ID </th>
-        <td mat-cell *matCellDef="let user"> {{user.userId |  slice:0:8 }}... </td>
+        <td mat-cell *matCellDef="let user"> {{user.externalId? user.externalId : ((user.userId | slice:0:8) + '...')}}</td>
       </ng-container>
       <ng-container matColumnDef="externalUserId">
         <th mat-header-cell *matHeaderCellDef mat-sort-header> External User ID </th>
-        <td mat-cell *matCellDef="let user"> {{user.externalUserId}} </td>
+        <td mat-cell *matCellDef="let user"> {{user.serviceUserId}} </td>
       </ng-container>
```

Comment:
Perhaps the external user ID does not even need to be shown here, instead the first header could be the actual user ID and the second column the external user ID.

Reply:
I'm confused! We have:
1) `userId`
2) `externalId`: (was `HumanReadableUserId` before)
3) `serviceUserId`: (was `externalUserId` before)
You mean we don't need `serviceUserId` in the table?

Reply:
I think we don't need the `serviceUserId` in the table, as long as users can see it in the info dialog.